lua/packer/plugin_types/git.lua

```diff
@@ -33,6 +33,21 @@ local function ensure_git_env()
   end
 end
 
+local breaking_change_pattern = [=[[bB][rR][eE][aA][kK][iI][nN][gG][ _][cC][hH][aA][nN][gG][eE]]=]
+local function mark_breaking_commits(plugin, commit_bodies)
+  local commits = vim.split(table.concat(commit_bodies, '\n'), '===COMMIT_START===')
+  for _, commit in ipairs(commits) do
```

Comment:
Using the `vim.gsplit()` iterator might be a little faster, since you don't have to loop over the string to collect it into a table first (I don't have any benchmarks to back this up though)
```suggestion
  local commits = vim.gsplit(table.concat(commit_bodies, '\n'), '===COMMIT_START===', true)
  for commit in commits do
```

I just checked this branch out locally, works very well! Love this feature already :+1:

Reply:
Thanks; this seems like a reasonable suggestion. Even if it's not faster, I don't think it will be slower, and it's a tad more idiomatic.